src/solver/aleatoire/alea_tirage_au_sort_chroniques.cpp

```diff
@@ -93,6 +93,27 @@ static void InitializeTimeSeriesNumbers_And_ThermalClusterProductionCost(
             ptchro.Eolien
               = (data.series.width != 1) ? (long)data.timeseriesNumbers[0][year] : 0; // zero-based
         }
+        // Renewable
+        {
+            auto end = area.renewable.list.mapping.end();
```

Comment:
to be checked if **mapping** and **cluster** list are redundant

Reply:
`mapping` is redundant. Removed in commit d9895347